Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -707,6 +727,11 @@ func printReplicationController(controller *api.ReplicationController, w io.Writ
 	name := controller.Name
 	namespace := controller.Namespace
 	containers := controller.Spec.Template.Spec.Containers
+	kind := options.KindName
```

Comment:
I will change `options.KindName` to `options.Kind` as it will better reflect its value